internal/extproc/translator/openai_awsbedrock.go

```diff
@@ -152,95 +166,183 @@ func (o *openAIToAWSBedrockTranslatorV1ChatCompletion) openAIToolsToBedrockToolC
 	return nil
 }
 
-// OpenAIMessageToBedrockMessage converts openai ChatCompletion messages to aws bedrock messages
-func (o *openAIToAWSBedrockTranslatorV1ChatCompletion) OpenAIMessageToBedrockMessage(openAIReq *openai.ChatCompletionRequest,
+// regDataURI follows the web uri regex definition.
+// https://developer.mozilla.org/en-US/docs/Web/URI/Schemes/data#syntax
+var regDataURI = regexp.MustCompile(`\Adata:(.+?)?(;base64)?,`)
```

Comment:
@wengyao04 can you help qualify the content type with `image/` ? we are only supporting image data url here.

Reply:
@yuzisun we have image format validation in https://github.com/envoyproxy/ai-gateway/pull/132/files#diff-6da52452dbe8387dcea4abc098618db9120d7d8b3071bf502d60530912158737R213-R225. Prefer to use data uri syntax for regex https://en.wikipedia.org/wiki/Data_URI_scheme